### ISSUE TITLE
review notifications on android

### ISSUE THREAD

**Issue by patte** [orbiting/app#101]
- [ ] notifications are not shown if app is in foreground
- [x] when having the Login overlay open and opening a new authorize notification, the Login overlay doesn't refresh with the new request, it stays where it is.

**Comment by diegomura** [comment]
@patte are you sending notifications with `priority: 'max'` for android? Because than can make the difference between showing the heads up and being a silent notification

**Comment by patte** [comment]
No, I'm using the default `high`. I'll test with `max` (WTF?) tomorrow and let you know.

**Comment by diegomura** [comment]
Yes. So confusing haha. Just let me know when you do!

**Comment by patte** [comment]
hey @diegomura! Where did you get `priority: 'max'` from? I'm not seeing that in the firebase docs: https://firebase.google.com/docs/cloud-messaging/concept-options#setting-the-priority-of-a-message
And just testing it out returns an error: `Raw server response: "Field "priority" must be a JSON number: max`

**Comment by patte** [comment]
I noticed that on the device I tested this before, the default setting is not to show the banner. I had to enable this in the settings. Now it's working fine in fore- and background. Sorry for the false alarm! Still good to have found out, I'll tell our careteam about it.
![screenshot_2018-08-01-22-57-44](https://user-images.githubusercontent.com/3500621/43548555-834077de-95de-11e8-8052-1eba61944ce6.png)
